.prettierrc.js

```diff
@@ -1,7 +0,0 @@
-/**
- * @prettier
- */
-module.exports = {
-  ...require("@stellar/prettier-config"),
-  requirePragma: true,
-};
```

Comment:
I think we should still keep this file and continue using Prettier. We can use pragmas to make sure it's only implemented on the pages we want:
https://prettier.io/docs/en/options.html#require-pragma

Reply:
Good catch! We're using `prettier.config.js` name in other projects, so I used that here as well, but I forgot to add `requirePragma` param.